yaks/easytest/easytest.cabal

```diff
@@ -1,8 +1,8 @@
+cabal-version: 2.2
```

Comment:
Should this be` >= 2.2` in each case?  Or what happens if you have a version that's too new?

Reply:
So there's a story here!

To get common stanzas we have to specify at least 2.2. But changed this field in two ways:

1. It has to be at the top of the cabal file (easy enough)

2. It has to be an exact version, not a range.

You can find more documentation on it here: https://cabal.readthedocs.io/en/latest/cabal-package.html#pkg-field-cabal-version

My understanding is that later versions of cabal-the-library will still work.

All that the `cabal-version` field is identifying is which version of the spec this particular cabal file complies with, but later versions of cabal-the-library can still read older versions of cabal-the-spec-for-cabal-files.

I checked, and I think that's borne out by my current setup:
```sh
$ stack exec -- cabal --version
cabal-install version 3.0.0.0
compiled using version 3.0.0.0 of the Cabal library
```

Reply:
I see, great!